src/extensions/curation/CurationSortingView/TabsView.tsx

```diff
@@ -41,7 +41,7 @@ const TabPanel: FunctionComponent<{index: number, currentTabIndex: number, width
 
 const TabsView: FunctionComponent<SortingViewProps & {width?: number, height?: number}> = (props) => {
     const classes = useStyles();
-    const [currentTabIndex, setCurrentTabIndex] = React.useState<number>(0);
+    const [currentTabIndex, setCurrentTabIndex] = React.useState<number>(1);
```

Comment:
For debugging I was starting at tab index=1, but this will get moved back to index 0